### PR TITLE
 Switch from deprecated terminal module to new terminal module.

### PR DESCRIPTION
**Pull request by evankanderson**
Switch from deprecated terminal module to replacement terminal module.

> SA1019: package golang.org/x/crypto/ssh/terminal is deprecated: this package moved to golang.org/x/term.  (staticcheck)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -20,7 +20,7 @@ require (
 	github.com/yudai/hcl v0.0.0-20151013225006-5fa2393b3552 // indirect
 	github.com/yudai/umutex v0.0.0-20150817080136-18216d265c6b // indirect
 	go.uber.org/zap v1.16.0
-	golang.org/x/crypto v0.0.0-20210415154028-4f45737414dc
+	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
```
Comment (embano1) [inline go.mod:23]: Any reason (beyond go mod) an older version of this pkg is used?

**Review by embano1 — APPROVED** [review]
Once checks pass, LGTM (just one question regarding term pkg version)

File: go.mod (hunk 1 above)
Comment (evankanderson) [reply go.mod:23]: I just ran `hack/update-deps.sh`, and trusted it.

Comment (evankanderson) [reply go.mod:23]: I'm not sure, because it looks like there are more recent commits in https://github.com/golang/term

Comment (embano1) [reply go.mod:23]: Feel free to upgrade if it makes sense:

```console
$ go get -u golang.org/x/term
```

```patch
diff --git a/go.mod b/go.mod
index c1ec9aee..f174007a 100644
--- a/go.mod
+++ b/go.mod
@@ -20,7 +20,8 @@ require (
 	github.com/yudai/hcl v0.0.0-20151013225006-5fa2393b3552 // indirect
 	github.com/yudai/umutex v0.0.0-20150817080136-18216d265c6b // indirect
 	go.uber.org/zap v1.16.0
-	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
+	golang.org/x/sys v0.0.0-20210423185535-09eb48e85fd7 // indirect
+	golang.org/x/term v0.0.0-20210422114643-f5beecf764ed
 	gotest.tools v2.2.0+incompatible
 	k8s.io/api v0.19.7
 	k8s.io/apimachinery v0.19.7

```

Comment (n3wscott) [reply go.mod:23]: ping on this update ^

**Comment by embano1** [comment]
Is this obsolet then @evankanderson ?

**Pull request closed without merging**